### DISCUSSION TITLE
Example code

### DISCUSSION THREAD

**Discussion by gr-qft** [tailwindlabs/tailwindcss#4827] [Help]
Hi! 

Thanks for this library and actually everything tailwindcss. Could you make the code for the example on https://tailwindcss-forms.vercel.app/ available? Thanks!

**Comment by darrylmorley** [comment]
Are you looking for this https://github.com/tailwindlabs/tailwindcss-forms/blob/master/index.html

**Reply by gr-qft** [reply]
Great, thanks for the link!
